src/plugins/data/server/index.ts

```diff
@@ -20,8 +20,36 @@
 import { PluginInitializerContext } from '../../../core/server';
 import { DataServerPlugin, DataPluginSetup, DataPluginStart } from './plugin';
 
+import {
```

Comment:
Do we need to export a separate namespace for both public/server or can we create the namespace inside the common folder and just re-export here and in public?

Reply:
We do need separate namespaces for public \ server as they have really different needs @lukeelmers